frontend/viewer/src/lib/services/projects-service.ts

```diff
@@ -50,7 +50,7 @@ export class ProjectService implements ICombinedProjectsService {
   async downloadCrdtProject(project: Project) {
     const r = await fetch(`/api/download/crdt/${project.server!.authority}/${project.code}`, {method: 'POST'});
     if (!r.ok) {
-      AppNotification.display(`Failed to download project ${project.code}: ${r.statusText} (${r.status})`, 'error', 'long');
+      AppNotification.display(`Failed to download project ${project.code}: ${r.statusText} (${r.status})`, 'error');
```

Comment:
any reason you made these no longer long?

Reply:
I don't remember deciding to do this, but it means it needs to be manually dismissed, which I think is generally the right choice for errors.